00-introduction.md

```diff
@@ -21,6 +21,7 @@ This is version 0.
 7. [BOLT #7](07-routing-gossip.md): P2P Node and Channel Discovery
 8. [BOLT #8](08-transport.md): Encrypted and Authenticated Transport
 9. [BOLT #9](09-features.md): Assigned Feature Flags
+11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
```

Comment:
10 is the DNS seed protocol, which I haven't finalized yet. I don't think that merging the bolts into BIPs is actually desirable since they are not directly Bitcoin related, rather they describe a protocol built on top of Bitcoin, or other cryptocurrency.